sheets-import/config.gs

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var TESTING = true;
+
+/**
+ * @TODO: Replace the following with the name of the add-on you are building
+ * and the title of your sidebar
+ */
+
+var ADDON_NAME = 'YOUR_ADDON_NAME_HERE';
+var SIDEBAR_TITLE = 'My Super Awesome Sidebar';
```

Comment:
Fun, but it's better to reference the template title in the sidebar title (such as "Import to Sheets") so that the developer is immediately aware of which add-on (or template in this case) created it. No need to include "Sidebar" in the title either, as that is immediately obvious when it is open.